geometry/convexHull.js

```diff
@@ -18,13 +18,16 @@ export const ConvexHull = (geometries) => {
 export const convexHull = (geometry, geometries) =>
   ConvexHull([geometry, ...geometries]);
 
-export const ChainConvexHull = (geometries) => {
+export const ChainConvexHull = (geometries, { close = false } = {}) => {
```

Comment:
Close links the last one to the first one in the chain?

Reply:
Yeah, the same works with Link() which makes Loop() redundant -- I should deprecate Loop.